unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NotificationMessagesHUD/ChatNotificationController.cs

```diff
@@ -80,23 +87,21 @@ private void HandleMessageAdded(ChatMessage message)
                         var privateModel = new PrivateChatMessageNotificationModel(message.messageId,
                             message.sender, message.body, message.timestamp, peerName, peerProfilePicture);
 
-                        mainChatNotificationView.AddNewChatNotification(privateModel);
-
-                        if (topNotificationPanelTransform.Get().gameObject.activeInHierarchy)
+                        if(message.sender != openedChat.Get())
                         {
-                            if(message.sender != openedChat.Get())
+                            mainChatNotificationView.AddNewChatNotification(privateModel);
+                            if (topNotificationPanelTransform.Get().gameObject.activeInHierarchy)
                                 topNotificationView.AddNewChatNotification(privateModel);
                         }
                         break;
                     case ChatMessage.Type.PUBLIC:
                         var publicModel = new PublicChannelMessageNotificationModel(message.messageId,
                             message.body, channel?.Name ?? message.recipient, channel?.ChannelId, message.timestamp, peerName);
 
-                        mainChatNotificationView.AddNewChatNotification(publicModel);
-
-                        if (topNotificationPanelTransform.Get().gameObject.activeInHierarchy)
+                        if(channel?.ChannelId != openedChat.Get())
                         {
-                            if(channel?.ChannelId != openedChat.Get())
+                            mainChatNotificationView.AddNewChatNotification(publicModel);
```

Comment:
what if you have the #nearby opened when you receive the message. Will you be able to see the notification after you close the window? I think it should be there, but you can confirm with product though.

Reply:
if you saw the message arriving you shouldn't also receive the notification, as it is for the top notification